airflow/www/templates/airflow/dag.html

```diff
@@ -366,9 +366,9 @@ <h4 class="modal-title" id="dagModalLabel">
     var task_id = '';
     var execution_date = '';
     var subdag_id = '';
-    var show_es_logs = false;
-    {% if show_external_logs is defined %}
-      show_es_logs = '{{ show_external_logs }}' == "True";
+    var show_remote_log_redirect = false;
```

Comment:
```suggestion
    var show_external_log_redirect = false;
```
This name is even more confusing here.